Code/BasicFilters/json/ShiftScaleImageFilter.json

```diff
@@ -91,6 +102,89 @@
           "value" : 0
         }
       ]
+    },
+    {
+      "tag" : "OutputPixel",
+      "description" : "Shift and scale with smaller output pixel type",
```

Comment:
Possibly say "smaller output range pixel type"? I'm not sure adding "range" is clearer. Modify if you think adding it makes the sentence clearer.